src/ElvContracts.js

```diff
@@ -305,7 +305,7 @@ class ElvContracts {
       path.resolve(__dirname, "../contracts/v4/Payment.abi")
     );
     const abistrToken = fs.readFileSync(
-      path.resolve(__dirname, "../contracts/v4/IERC20.abi")
+      path.resolve(__dirname, "../contracts/v4/ElvToken.abi")
```

Comment:
@elv-preethi I changed this ABI because the original IERC20 doesn't have decimals (so we fail to read decimals without this change).  We could use another interface there if there is an extended one with decimals.